src/views/RegisterUser.vue

```diff
@@ -151,27 +151,68 @@ export default defineComponent({
           },
           confirm: {
             requiredValidator,
-
-            sameAs: sameAs(state.password.password),
+            sameAs: sameAs(state.formData.password.password),
           },
         },
         registrationKey: {
           requiredValidator,
-          keyValidator,
         },
       };
     });
-    const v$ = useVuelidate(rules, state);
+    const v$ = useVuelidate(rules, state.formData);
+    async function sendInfo() {
+      const userData: UserStateInterface['Data'] = {
+        registration_token: state.formData.registrationKey,
+        first_name: state.formData.firstName,
+        last_name: state.formData.lastName,
+        email: state.formData.email,
+        password: state.formData.password.confirm,
+        contacts: [{ id: Date.now(), type: 'email', main: false, email: state.formData.email }],
+      };
+      store.dispatch('userStore/SET_USER_INFO', userData);
```

Comment:
missing await for async fn call
vuex actions are async, composition API needs manual async handlers